### PR TITLE
VAL-607: Fix forta deployment process and provide documentation

### PR DESCRIPTION
**Pull request by AlexandrMov**
- downgrade the `packagemanager` to be able to build docker image
- deployment process described

### CODE REVIEW

File: ethereum-validators-set/package.json [hunk 1]
```diff
@@ -42,5 +43,5 @@
     "ts-jest": "^29.1.1",
     "typescript": "^4.3.4"
   },
-  "packageManager": "yarn@3.3.0"
+  "packageManager": "yarn@1.22.21"
```
Comment (F4ever) [inline ethereum-validators-set/package.json:46]: It was downgraded?

File: ethereum-validators-set/README.md [hunk 2]
```diff
@@ -148,3 +148,29 @@ For example, you need to add `testnet` tier support for some new sub-agent:
...
+
+```json
+{
+  "jsonRpcUrl": "<jsonRpcUrl>",
+  "keyfilePassword": "<passphrase from the wallet>",
+  "agentId": "0x399792adf7b57c0adc8de54ee27284d8ef70b0af2ff3c2c97f69ae894f66759a"
+}
+```
+
+Beside that, place into `~/.forta` directory the private key with name: `UTC--2024-01-31T01:37:55.270Z--123.json`
```
Comment (F4ever) [inline ethereum-validators-set/README.md:168]: Maybe name should be somehow generated

Comment (AlexandrMov) [reply ethereum-validators-set/README.md:168]: yes, it's generated, once a new bot is created. But we use already existing one.

File: ethereum-validators-set/package.json (hunk 1 above)
Comment (AlexandrMov) [reply ethereum-validators-set/package.json:46]: Yes, newer version does not work: I couldn't push a new image with that version.

**Review by F4ever — APPROVED** [review]

**Pull request merged**